internal/constants/constants.go

```diff
@@ -66,3 +67,26 @@ const (
 	// UUIDSize is the length of a UUID, in bytes.
 	UUIDSize = 16
 )
+
+// Domain state
+const (
```

Comment:
Current convention in this package is to create a new type.

```go
type DomainState int
```

Then each constant will be prefixed with `DomainState`.